src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,170 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+### Hierarchy
+
+Each span can be connected to other spans using a parent-child hierarchy. 
+
+For example, a span called `updateJiraTask` would have child spans like `Check permissions`, `Update Task`, `sql.update`, `send.webhooks`.
+
+Each sub-task would have an id in the attribute `parent_span_id` id of the “updateTask” span.
```

Comment:
```suggestion
Each sub-task would have an id in the attribute `parent_span_id` id of the `updateJiraTask` span.
```

Reply:
@markstory what would be the appropriate slug?